package-lock.json

```diff
@@ -0,0 +1,17946 @@
+{
```

Comment:
I think the commit of the update lock shall be squashed with the commit updating the `packages.lock`